packages/manager-components/src/components/navigation/menus/action/action.component.tsx

```diff
@@ -28,15 +30,47 @@ export interface ActionMenuItem {
   target?: OdsHTMLAnchorElementTarget;
   onClick?: () => void;
   label: ReactI18NextChild | Iterable<ReactI18NextChild>;
+  actionIam?: string;
```

Comment:
```suggestion
  iamAction?: string;
```

Reply:
what do you thing about  iamActions?: string[]' cause the api parameters take an array